src/pages/demo.js

```diff
@@ -180,7 +240,42 @@ function Demo() {
             <DocSearchV2 {...project} />
           </ErrorBoundary>
         ) : (
-          <DocSearch {...project} />
+          <div className="uil-ph-32 uil-ta-center uil-d-flex uil-fxd-column">
+            <Button
+              onClick={onOpen}
+              className="uil-mt-16 uil-mb-16 uil-m-auto"
+              tag="button"
+              type="submit"
+              id="searchButton"
+              primary
+            >
+              Open modal to search on {project.indexName}
+            </Button>
+            {isOpen &&
+              createPortal(
+                <DocSearchModal
+                  {...project}
+                  navigator={{
```

Comment:
For links to effectively work, we need to remove `navigator`, `transformItems` and `hitComponent`.

Reply:
That was my next question. Should be done